services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -236,6 +244,7 @@ export const SideBar: React.FC<{ className: string; toggleSidebar: () => void }>
                 deleteAllActions();
             });
         handleClose();
+        updateActionsNumber(0);
```

Comment:
the `updateActionsNumber` is a little overcomplicated. what it actually does is update the sidebar _title_. 

the function name should indicate what the function does or its purpose. e.g. `updateSidebarTitle`